### PR TITLE
Add NollySenti Bitext Mining (MINERS)

### PR DESCRIPTION
**Pull request by gentaiscool**
NollySenti is Nollywood movie reviews for five languages widely spoken in Nigeria (English, Hausa, Igbo, Nigerian-Pidgin, and Yoruba.

## Checklist

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 

### Adding datasets checklist

**Reason for dataset addition**: ... 

- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`.

### CODE REVIEW

**Comment by gentaiscool** [comment]
@KennethEnevoldsen I created a new PR for NollySenti.

**Comment by gentaiscool** [comment]
I have completed the checklist @KennethEnevoldsen.

File: mteb/tasks/BitextMining/multilingual/NollySentiBitextMining.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        @inproceedings{shode2023nollysenti,
+            title={NollySenti: Leveraging Transfer Learning and Machine Translation for Nigerian Movie Sentiment Classification},
+            author={Shode, Iyanuoluwa and Adelani, David Ifeoluwa and Peng, Jing and Feldman, Anna},
+            booktitle={Proceedings of the 61st Annual Meeting of the Association for Computational Linguistics (Volume 2: Short Papers)},
+            pages={986--998},
+            year={2023}
+        }
+        """,
+        n_samples={"train": 1640},
+        avg_character_length={"train": 4.46},
```
Comment (KennethEnevoldsen) [inline mteb/tasks/BitextMining/multilingual/NollySentiBitextMining.py:48]: a bit short?

**Review by KennethEnevoldsen — COMMENTED** [review]
Will you add points and push models results as well?

**Comment by gentaiscool** [comment]
Thank you for the review. I addressed them. I updated the avg number of character per sample, added the results, and points. The new languages are `hau-Latn`, `ibo-Latn`, `pcm-Latn`, `yor-Latn`, so the points are 2 + 4 x 4 = 18. Plus, I added the point for the reviewer. @KennethEnevoldsen

File: mteb/tasks/BitextMining/multilingual/NollySentiBitextMining.py (hunk 1 above)
Comment (gentaiscool) [reply mteb/tasks/BitextMining/multilingual/NollySentiBitextMining.py]: I updated the avg character length per sample (previously I put the avg char per word)

**Pull request merged**